ledger-core/application/api/availability_test.go

```diff
@@ -39,7 +40,12 @@ func waitForStatus(t *testing.T, nc *api.NetworkChecker, expected bool) {
 }
 
 func TestAvailabilityChecker_UpdateStatus(t *testing.T) {
-	instestlogger.SetTestOutputWithIgnoreAllErrors(t)
+	instestlogger.SetTestOutputWithErrorFilter(t, func(s string) bool {
+		expectedError := strings.Contains(s, "no response or bad StatusCode") ||
+			strings.Contains(s, "Can't decode body: invalid character '}'") ||
+			strings.Contains(s, "connection refused")
```

Comment:
maybe move those strings to constants?

Reply:
what do you mean?

Reply:
they are not constants. That couple of expected errors that are in this test